src/sage/misc/remote_file.py

```diff
@@ -17,8 +16,8 @@ def get_remote_file(filename, verbose=True):
 
     OUTPUT:
 
-    creates a file in the temp directory and returns the absolute path
-    to that file.
+    This creates a file in the temp directory and returns the absolute path
+    to that file as a ``Path`` object.
 
```

Comment:
We can use ``` :class:`Path` ``` everywhere. Then we have a nice link.

Reply:
done, let's check if this works